ci-scripts/request-translation.sh

```diff
@@ -18,13 +18,16 @@ export EN_PATH="${TARGET_PATH}en-us"
 export JA_PATH="${TARGET_PATH}ja-jp"
 
 export TOPIC_BRANCH="docs_l10n_request_${TARGET^^}_$(date +"%s")"
-
 export RESOURCE_PATHS=( "Content/Resources" "Resources" "Data" "Skins" "Sitemap.xml" )
 
+if [[ ${TARGET} == "oce" || ${TARGET} == "oie" ]]; then
```

Comment:
`wf`, `asa`, and `oag` also contain `Content/Topics/ReleaseNotes` folders. The only product of the six main ones here that doesn't have such folder is `eu`.
Maybe we could use a conditional? If `Content/Topics/ReleaseNotes` present, move it, else no-op? 
Or, alternatively. check if target is `eu` then do nothing, else move the folder (present in all other targets).

Reply:
for those products release notes are in separate folder from other topics, and we can configure XTM project to translate folder with topics only.
asa, use `asa/en-us/Content/Topics/Adv_Server_Access` path:
<img width="622" alt="image" src="https://user-images.githubusercontent.com/4158731/210641439-3553ff85-1b26-44b3-875c-4a216b9f9304.png">
wf:
<img width="659" alt="image" src="https://user-images.githubusercontent.com/4158731/210641511-2af8cfda-c39b-4152-b1fe-249e85b8d6f8.png">
oag:
<img width="656" alt="image" src="https://user-images.githubusercontent.com/4158731/210641564-7c63dd03-3a9d-4ac6-9083-6e53b9f2a842.png">

only for `oie` and `oce` release notes are in same folder with other topics and we can't configure XTM.

Reply:
ah, i see, got it

Reply:
On the other hand moving release notes all the time looks simpler from coding stand point.

Reply:
let's double check with @pengyao-okta before merging. I didn't want to move files around when it's not required. But if we're doing it for 2/6 targets, we can do it for 5/6 as well